account/tests.py

```diff
@@ -0,0 +1,155 @@
+        user_type=PROFESSIONAL_TYPE,
+        phone_number=PHONE_NUMBER,
+        country=COUNTRY,
+        city=CITY,
+        address=ADDRESS
+    )
+    return profile
+
+
+@pytest.fixture
+def professional(profile):
+    professional_profile = profile
+    professional_profile.user_type = PROFESSIONAL_TYPE,
+    professional = Professional.objects.create(
+        profile_id=professional_profile,
+        profession=PROFESSION,
+        description=DESCRIPTION
+    )
+    return professional
+
+
+@pytest.fixture
+def client(profile):
+    client_profile = profile
+    client_profile.user_type = CLIENT_TYPE,
+    client = Client.objects.create(
+        profile_id=profile,
+        birthday=BIRTHDAY
+    )
+    return client
+
+
+@pytest.mark.django_db
+class TestProfileModel:
+    def test_new_profile(self, profile):
+        assert profile.user_type == PROFESSIONAL_TYPE
+        assert profile.phone_number == PHONE_NUMBER
+        assert profile.country == COUNTRY
+        assert profile.city == CITY
+        assert profile.address == ADDRESS
+
+    def test_get_profile(self, profile):
+        profile.user_id.save()
+        profile.save()
+        assert profile in Profile.objects.all()
+
+    def test_delete_profile(self, profile):
+        profile.user_id.save()
+        profile.save()
+        profile.delete()
+        assert profile not in Profile.objects.all()
+
+    def test_delete_user_deletes_profile(self, profile):
+        profile.user_id.save()
+        profile.save()
+        profile.user_id.delete()
+        assert profile not in Profile.objects.all()
+
+
+@pytest.mark.django_db
+class TestProfessionalModel:
+    def test_new_professional(self, professional):
+        assert professional.profession == PROFESSION
+        assert professional.description == DESCRIPTION
+
+    def test_get_professional(self, professional):
+        professional.profile_id.user_id.save()
+        professional.profile_id.save()
+        professional.save()
+        assert professional in Professional.objects.all()
+
+    def test_delete_professional(self, professional):
+        professional.profile_id.user_id.save()
+        professional.profile_id.save()
+        professional.save()
+        professional.delete()
+        assert professional not in Professional.objects.all()
+
+    def test_delete_user_deletes_professional(self, professional):
+        professional.profile_id.user_id.save()
+        professional.profile_id.save()
+        professional.save()
+        professional.profile_id.delete()
+        assert professional not in Professional.objects.all()
+
+
+@pytest.mark.django_db
+class TestClientModel:
+    def test_new_client(self, client):
+        assert client.birthday == BIRTHDAY
+
+    def test_get_client(self, client):
+        client.profile_id.user_id.save()
+        client.profile_id.save()
+        client.save()
+        assert client in Client.objects.all()
+
+    def test_delete_professional(self, client):
```

Comment:
Is it _proffessional or client? the tets name could be wrong
same goes all the others

Reply:
its need to be a client of course,
I fixed it
and once a profile is deleted the client/professional is deleted 
about the other side I prefer to open a new ticket for that 
I think that this PR is too fat :)